satip/app.py

```diff
@@ -81,8 +91,8 @@
     type=click.BOOL,
 )
 @click.option(
-    "--use-backup",
-    envvar="USE_BACKUP",
+    "--use-hr-serviri",
+    envvar="USE_HR_SERVIRI",
```

Comment:
So is this the same service as the backup, but we're just being more explicit?

Reply:
yea exactly, I was getting confused, so I wanted to write it properly